input/fsh/SD_Procedures.fsh

```diff
@@ -1,79 +1,155 @@
+Description: "A summary of delivered teleradiotherapy treatment. The scope is a prescription consisting of one or multiple Fractions. A prescription delivery instance should end when there is a change in the target volume of a body site, treatment fraction size, modality, or treatment technique.
+Conformance statement:  TBD"
+* insert RadiotherapyPrescriptionDeliveryRS
+* code from TeleradiotherapyModalityVS (required)
+* code ^short = "Modality"
+* code ^definition = "The modality (radiation type) for the external beam procedure."
+* code obeys teleradiotherapy-procedure-code-invariant
+* extension[radiotherapyTechnique].valueString from TeleradiotherapyTechniqueVS (extensible)
+* partOf only Reference(RadiotherapyCourseSummary)
+* usedCode from TeleradiotherapyDeviceVS (extensible)
 
-Conformance statement:
 
-Procedure resources associated with an mCODE patient with Procedure.category SNOMED-CT 387713003 MAY conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patient that would reasonably be expected to conform to this profile SHOULD be published in this form. Specifically, we expect that any surgical procedure related to the treatment of a `PrimaryCancerCondition` or `SecondaryCancerCondition` would be published in this form."
-* insert CancerRelatedProcedureCommonRuleSet
-// Do not insert the category slicing rules because Procedure.category is 0..1.
-* category = SCT#387713003 //"Surgical procedure"
-* code from CancerRelatedSurgicalProcedureVS (extensible)
+    Invariant: teleradiotherapy-procedure-code-invariant
+    Description: "If the code 'Other Teleradiotherapy Modality, specify' is used, a second code from outside the original value set must be present."
+    Expression: "coding.where(code = 'OtherTeleradiotherapyModality').exists() implies coding.where(code != 'OtherTeleradiotherapyModality' and $this.memberOf('http://hl7.org/fhir/us/mcode/ValueSet/mcode-teleradiotherapy-modality-vs').not()).exists()"
+    Severity:   #error
 
-/*
-Extension: RadiationDose
-Id: mcode-radiation-dose
-Title:  "Radiation Dose"
-Description: "Information related to the dose of radiation received in a RadiationProcedure, including the dose per fraction, the number of fractions delivered, and the total radiation dose delivered."
-* value[x] 0..0
-* extension contains
-    RadiationDosePerFraction 0..1 and
-    RadiationFractionsDelivered 0..1 and
-    TotalRadiationDoseDelivered 0..1
+Profile:  BrachytherapyPrescriptionDelivery
+Parent:   USCoreProcedure
+Id:       brachytherapy-prescription-delivery
+Title:    "Brachytherapy Prescription Delivery"
+Description: "A summary of delivered brachytherapy treatment. The scope is a prescription consisting of one or multiple fractions. A new prescription delivery begins when there is a change in the target volume of a body site, treatment fraction size, modality, or treatment technique.Conformance statement: TBD"
```

Comment:
Typo fix:

```suggestion
Description: "A summary of delivered brachytherapy treatment. The scope is a prescription consisting of one or multiple fractions. A new prescription delivery begins when there is a change in the target volume of a body site, treatment fraction size, modality, or treatment technique.

Conformance statement: TBD"
```

Reply:
Fix later when we figure out the conformance